src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -131,34 +141,35 @@ def serialize_users(
     def get_from_group(self, group: Group) -> List[APIUser]:
         return [
             UserService.serialize_user(u)
-            for u in self.__base_user_query().filter(
+            for u in self._base_query().filter(
                 sentry_orgmember_set__organization=group.organization,
                 sentry_orgmember_set__teams__in=group.project.teams.all(),
                 is_active=True,
             )
         ]
 
-    def get_many(self, user_ids: Iterable[int]) -> List[APIUser]:
-        query = self.__base_user_query().filter(id__in=user_ids)
-        return [UserService.serialize_user(u) for u in query]
+    # def get_many(self, user_ids: Iterable[int]) -> List[APIUser]:
+    #     query = self._base_query().filter(id__in=user_ids)
+    #     return [UserService.serialize_user(u) for u in query]
 
     def get_from_project(self, project_id: int) -> List[APIUser]:
         try:
             project = Project.objects.get(id=project_id)
         except Project.DoesNotExist:
             return []
-        return self.get_many(project.member_set.values_list("user_id", flat=True))
+        return self.get_many(
+            filter=dict(user_ids=project.member_set.values_list("user_id", flat=True))
```

Comment:
Having a dedicated `filter` parameter that uses typed dicts is a nice solution. I prefer the ergonomics of that over having to build a request object for each operation.